src/os/ObjectStore.h

```diff
@@ -1378,26 +1378,27 @@ class ObjectStore {
     }
 
     /**
-     * Give the collection a hint.
-     *
-     * @param cid  - collection id.
-     * @param type - hint type.
-     * @param hint - the hint payload, which contains the customized
-     *               data along with the hint type.
-     */
+    * Give the collection a hint.
+    *
+    * @param cid  - collection id.
+    * @param type - hint type.
+    * @param hint - the hint payload, which contains the customized
+    *               data along with the hint type.
+    */
     void collection_hint(const coll_t& cid, uint32_t type, const bufferlist& hint) {
       if (use_tbl) {
-        __u32 op = OP_COLL_HINT;
-        ::encode(op, tbl);
-        ::encode(cid, tbl);
-        ::encode(type, tbl);
-        ::encode(hint, tbl);
-      } else {
-        Op* _op = _get_next_op();
-        _op->op = OP_COLL_HINT;
-        _op->cid = _get_coll_id(cid);
-        _op->hint_type = type;
-        ::encode(hint, data_bl);
+	__u32 op = OP_COLL_HINT;
+	::encode(op, tbl);
+	::encode(cid, tbl);
+	::encode(type, tbl);
+	::encode(hint, tbl);
+      }
+      else {
+	Op* _op = _get_next_op();
+	_op->op = OP_COLL_HINT;
+	_op->cid = _get_coll_id(cid);
+	_op->hint_type = type;
+	::encode(hint, data_bl);
```

Comment:
this hunk probably doesn't need to be here?  is it just fixing whitespace or something?